Index: source/Irrlicht/MacOSX/CIrrDeviceMacOSX.mm
===================================================================
--- source/Irrlicht/MacOSX/CIrrDeviceMacOSX.mm	(revision 5652)
+++ source/Irrlicht/MacOSX/CIrrDeviceMacOSX.mm	(working copy)
@@ -495,8 +495,8 @@
 		if(!CreationParams.WindowId) //load menus if standalone application
 		{
 			[[NSAutoreleasePool alloc] init];
-			[NSApplication sharedApplication];
-			[NSApp setDelegate:(id<NSFileManagerDelegate>)[[[AppDelegate alloc] initWithDevice:this] autorelease]];
+			[[NSApplication sharedApplication] activateIgnoringOtherApps:YES];
+			[NSApp setDelegate:(id<NSApplicationDelegate>)[[[AppDelegate alloc] initWithDevice:this] autorelease]];
 			[NSBundle loadNibNamed:@"MainMenu" owner:[NSApp delegate]];
 			[NSApp finishLaunching];
 		}
