Index: source/Irrlicht/MacOSX/CIrrDeviceOSX.mm
===================================================================
--- source/Irrlicht/MacOSX/CIrrDeviceOSX.mm	(revision 5652)
+++ source/Irrlicht/MacOSX/CIrrDeviceOSX.mm	(working copy)
@@ -495,8 +495,8 @@
 		if (!CreationParams.WindowId)
 		{
 			[[NSAutoreleasePool alloc] init];
-			[NSApplication sharedApplication];
-			[NSApp setDelegate:(id<NSFileManagerDelegate>)[[[CIrrDelegateOSX alloc] initWithDevice:this] autorelease]];
+			[[NSApplication sharedApplication] activateIgnoringOtherApps:YES];
+			[NSApp setDelegate:(id<NSApplicationDelegate>)[[[CIrrDelegateOSX alloc] initWithDevice:this] autorelease]];
             
             // Create menu
             
